src/host/_stream.cpp

```diff
@@ -769,7 +769,10 @@ using Microsoft::Console::VirtualTerminal::StateMachine;
             // See GH:12735, MSFT:31748387
             if (screenInfo.HasAccessibilityEventing())
             {
-                screenInfo.NotifyAccessibilityEventing(CursorPosition.X, CursorPosition.Y, CursorPosition.X + 1, CursorPosition.Y);
+                if (IConsoleWindow* pConsoleWindow = ServiceLocator::LocateConsoleWindow())
+                {
+                    LOG_IF_FAILED(pConsoleWindow->SignalUia(UIA_Text_TextChangedEventId));

```

Comment:
I'm very surprised that this works! Narrator was very bad about detecting what changed, so this is a good thing to see working.